### ISSUE TITLE
pip3 install biopython; fails on ubuntu:14.04

### ISSUE THREAD

**Issue by CptApature** [biopython/biopython#1600]
### Setup

Since the release of biopython 1.71 I am unable to `pip install` the package with python3.4 on ubuntu14..
I'm running a clean ubuntu:14.04 docker image: `docker run -it ubuntu:14.04`

```bash
apt-get update
apt-get install -y python3-pip
pip3 install biopython==1.71
...
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 7116: ordinal not in range(128)
```
I don't have this issue with either ubuntu:14.04+python2.7 or on ubuntu:16.04+(python2.7, python 3.5).
biopython==1.70 also proceeds without error on all systems

### Expected behaviour
Install succeeds

### Actual behaviour

Full exception message: 
```
$ pip3 install biopython==1.71
Downloading/unpacking biopython
  Downloading biopython-1.71.tar.gz (16.0MB): 16.0MB downloaded
  Running setup.py (path:/tmp/pip_build_root/biopython/setup.py) egg_info for package biopython
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/tmp/pip_build_root/biopython/setup.py", line 435, in <module>
        readme_rst = handle.read()
      File "/usr/lib/python3.4/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 7116: ordinal not in range(128)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/tmp/pip_build_root/biopython/setup.py", line 435, in <module>

    readme_rst = handle.read()

  File "/usr/lib/python3.4/encodings/ascii.py", line 26, in decode

    return codecs.ascii_decode(input, self.errors)[0]

UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 7116: ordinal not in range(128)

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /tmp/pip_build_root/biopython
Storing debug log for failure in /root/.pip/pip.log
```

### Steps to reproduce

See above

**Comment by peterjc** [comment]
Possible workaround:

```
$ LANG="en_US.UTF-8" pip3 install biopython==1.71
```

**Comment by CptApature** [comment]
default encoding is utf-8
``` bash
$ python3 -c 'import sys; print(sys.version); print(sys.getdefaultencoding())'
3.4.3 (default, Nov 28 2017, 16:41:13) 
[GCC 4.8.4]
utf-8
```
Setting `LANG="en_US.UTF-8"` had no effect for me

**Comment by peterjc** [comment]
There are multiple ways to set the locale/encoding... do you know what language your Ubuntu is set too? That would help to estimate how many more people would be affected.

As an aside, your pip looks like it is going to compile from source as it downloaded ``biopython-1.71.tar.gz`` rather than a wheel file - which is odd as we provide one:

https://pypi.python.org/packages/55/63/795ac9b613c2e9d3c406589e5dec628f90bcca3d5acb73b005be1df47714/biopython-1.71-cp34-cp34m-manylinux1_x86_64.whl

Could you try downloading that by hand, and then doing:

```
$ pip3 install biopython-1.71-cp34-cp34m-manylinux1_x86_64.whl
```

**Comment by CptApature** [comment]
I'm in the Docker Ubuntu:14.04 base image; so there is no locale set: https://www.thomas-krenn.com/en/wiki/Configure_Locales_in_Ubuntu#No_locale_set 

Generating a en_US.UTF-8 lcoale and using $LANG seems to fix the problem:
```bash
apt-get update && apt-get install python3-pip
pip3 install biopython # This fails
locale-gen LANG="en_US.UTF-8"
pip3 install biopython # This fails too
LANG="en_US.UTF-8" pip3 install biopython==1.71 # Bug resolved 
```
To be honest; my understanding of locales in linux is pretty non-existent so I'm not sure what's really going on under the hood here.


The wheel you linked to fails to install
```bash 
# pip3 install biopython-1.71-cp34-cp34m-manylinux1_x86_64.whl 
      biopython-1.71-cp34-cp34m-manylinux1_x86_64.whl is not a supported wheel on this platform.
      Storing debug log for failure in /root/.pip/pip.log
```
Full log:
```
biopython-1.71-cp34-cp34m-manylinux1_x86_64.whl is not a supported wheel on this platform.
Exception information:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pip/basecommand.py", line 122, in main
    status = self.run(options, args)
  File "/usr/lib/python3/dist-packages/pip/commands/install.py", line 257, in run
    InstallRequirement.from_line(name, None))
  File "/usr/lib/python3/dist-packages/pip/req.py", line 168, in from_line
    raise UnsupportedWheel("%s is not a supported wheel on this platform." % wheel.filename)
pip.exceptions.UnsupportedWheel: biopython-1.71-cp34-cp34m-manylinux1_x86_64.whl is not a supported wheel on this platform.
```

**Comment by peterjc** [comment]
Thanks for the update.

Good news that you could get this to work by setting a locale. Bad news that this issue will likely affect other many people if they use the same Docker image.

Separately, on the wheel issue, I'm trying to find an authoritative source on the oldest Ubuntu/Debian to support manylinux1.

**Comment by CptApature** [comment]
I don't have an authoritative source, but given that when the manylinux PEP513 was created (https://www.python.org/dev/peps/pep-0513/#the-manylinux1-policy) Ubuntu 14.04 was the latest stable release of Ubuntu I'd be surprised if it was unsupported! 

I checked the deb packages listed on the PEP and ensured all of them were installed and updated (some weren't actually installed in the base image, but were available), but still had the same issues. 

I've checked it on a CentOS 5.11 image and it installs fine under python3.4

**Comment by peterjc** [comment]
Thank you for reporting this and helping to understand the issue.

This was fixed in Biopython 1.72, and the work in pull request #1607 should ensure our ``README.rst`` file is plain ASCII and thus avoid a accidental recurrence of this problem.

(We can later relax that to ``latin1`` or even ``utf-8`` as needed as long as we make the encoding explicit)
